README.md

```diff
@@ -136,7 +136,7 @@ use $ATOMs; by offering what is misleadingly referred to as "liquid staking".
 
 ## Role as $ATOM "Liquid Staking" Provider
```

Comment:
In this section we need to be careful with the renaming of $phATOM to $PHOTON.

This section talks about the "$ATOM-version" of $PHOTON. We should disambiguate if we want to use the same name while we work on the final design/naming scheme.

Reply:
Thx, if you spot additional ones, feel free to share.

I’ll fix and make a second review soon.